### ISSUE TITLE
Can't authenticate with Twitter in the docker instance

### ISSUE THREAD

**Issue by juananpe** [renatolond/mastodon-twitter-poster#885]
I have installed and configured a Docker instance (my .env file includes correct TWITTER_CLIENT_ID and TWITTER_CLIENT_SECRET values). I can log-in into http://localhost:1081 and authorize my Mastodon account but whenever I click on the Twitter authorization button I get this error:

<img width="1260" alt="image" src="https://user-images.githubusercontent.com/1078305/203781427-d7668443-3116-4cc2-aa88-f321eb44402e.png">

When tailing my logs I can see this error:

I, [2022-11-24T12:10:32.682200 #1]  INFO -- : [37acf058-53aa-412f-8bbb-fd8608cf6f91] Started POST "/users/auth/twitter" for 172.24.0.1 at 2022-11-24 12:10:32 +0000
I, [2022-11-24T12:10:32.682479 #1]  INFO -- omniauth: (twitter) Request phase initiated.
F, [2022-11-24T12:10:32.940458 #1] FATAL -- : [37acf058-53aa-412f-8bbb-fd8608cf6f91]
[37acf058-53aa-412f-8bbb-fd8608cf6f91] OAuth::Unauthorized (401 Unauthorized):

Does it mean that the TWITER_CLIENT_ID or CLIENT_SECRET are invalid? How can increase the debug level to know more about the reason that is causing this error? Thanks.

**Comment by renatolond** [comment]
Does this comment solve it for you? https://github.com/renatolond/mastodon-twitter-poster/issues/827#issuecomment-1304501354

**Comment by juananpe** [comment]
I haven't access to the instance now, but I'm sure that your suggestion will work. My bad. Thanks!

**Comment by juananpe** [comment]
MMmhh... strange. I have updated my credentials to use the Consumer Key, Consumer Secret but it is still showing same behavior. I tried to create a new app in Twitter and the button to create one was disabled. So I think that I have to probably apply for a new [Developer Account](https://developer.twitter.com/en/apply-for-access) (I was trying to set-up a docker instance of the cross-poster recycling an old twitter-app... ) I'll ping this thread again when I get new information (my Developer Account request approved)
